conda_gitenv/tests/integration/setup_samples.py

```diff
@@ -23,13 +23,15 @@ def add_env(repo, name, spec):
     return branch
 
 
-def update_env(repo, branch, spec):
+def update_env(repo, branch, spec, comment=None):
```

Comment:
@corinnebosley It's used by the new integration test that I created, so that the caller can add a custom log message to a repo commit.

I also suspect that my new integration test is not thread safe and is causing the travis to complain ... which I need to look in to. The integration test work fine standalone, but sadly not in travis :cry: